packages/mui-styled-engine/src/GlobalStyles/GlobalStyles.d.ts

```diff
@@ -8,4 +8,4 @@ export interface GlobalStylesProps<Theme = {}> {
 
 export default function GlobalStyles<Theme = {}>(
   props: GlobalStylesProps<Theme>,
-): React.ReactElement<any>;
+): React.ReactElement<unknown>;
```

Comment:
@siriwatknp what do you think this type should be?

Reply:
I think it should be `React.JSX.Element`:

https://www.typescriptlang.org/play/?ts=4.1.5#code/JYWwDg9gTgLgBAJQKYEMDG8BmUIjgcilQ3wG4BYAKCrQgDsBneAFSSbgF44AKMHMBgC44Abzgw2MYUyjA6AczgBfAJScAfKKpwdcIjACuUOnAA8AE2AA3OAHp1VJVWfVXVU63YSmHAES+7dVIgA